docs/install/remote.md

```diff
@@ -91,6 +91,31 @@ mlrun.set_environment("http://localhost:8080", artifact_path="./")
 mlrun.set_environment("<remote-service-url>", access_key="xyz", username="joe")
 ```
 
+3. Run `mlrun config` command line to get (default) or set the environment variables into the default environment file. It will store all the configuration into the default environment file, without a need to edit your own environment file.
+
+    Supported commands:
+        `get` (default) - list the local or remote configuration
+                        (can specify the remote api + credentials or an env_file)
+        `set`           - set configuration parameters in mlrun default or specified .env file
+        `clear`         - delete the default or specified config .env file
+
+    The `set` command can work with the following parameters:
+    `-a` for setting the url (local or remote) for MLRun API 
```

Comment:
`-a` to set the url (local or remote) for MLRun API 
    `-p` to set the artifact path (optional)
    `-u` to set the username (for remote access)
    `-k` to set the access key (for remote access)
    `-e` to set the environment variables (optional)